v4/export/writer_util_test.go

```diff
@@ -140,6 +144,20 @@ func (s *testUtilSuite) TestWriteInsertInCsv(c *C) {
 		"3;'male';'john@mail.com';'020-1256';'healthy'\n" +
 		"4;'female';'sarah@mail.com';'020-1235';'healthy'\n"
 	c.Assert(bf.String(), Equals, expected)
+
+	// test delimiter that included in values
+	bf.Reset()
+	opt.separator = []byte(",")
```

Comment:
Separator should be multi-byte too

Reply:
addressed in https://github.com/pingcap/dumpling/pull/116/commits/bd8134310936180d35dfeedb2ac5d1b78c9446c1